composer.json

```diff
@@ -14,7 +14,8 @@
         "symfony/framework-bundle" : "~3.0"
     },
     "require-dev" : {
-        "atoum/atoum" : "master-dev"
+        "atoum/atoum" : "master-dev",
+        "symfony/yaml": "~3.0"
```

Comment:
no need to add this here because `symfony/framework-bundle` already include this package => https://packagist.org/packages/symfony/framework-bundle